tools/excelRTD/ExcelRTD.mpc

```diff
@@ -9,7 +9,7 @@ project(ExcelRTD): taolib, dcpslib, dcps_inforepodiscovery, dcps_tcp, dcps_monit
     compile_flags += /bigobj
   }
 
-  specific (vc10, vc11, vc12, vc14) {
+  specific (vc10, vc11, vc12, vc14, vs2017) {
```

Comment:
Could this not be `specific(prop:microsoft)`

Reply:
Well it's a Excel plugin so the whole thing is Windows specific, but I got a different idea:

The README says we support MSVC 9, which I'm guessing would be the current reason for the else case but I'm thinking maybe flip the expression:

```
specific (vc9) {
  macros += RTD_DLL_NAME=<%equote%>$(TargetFileName)<%equote%>
} else {
  macros += RTD_DLL_NAME=<%quote%>$(TargetFileName)<%quote%>
}
```

Reply:
We can also drop vc9 support for this feature.